### PR TITLE
Issues 1954

### PR DESCRIPTION
**Pull request by raftaar1191**
## Description
PR for issue #1954 

## How Has This Been Tested?
Manually tested 

## Types of changes
Bug fix (non-breaking change which fixes an issue)

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style.
- [ ] My code follows has proper inline documentation.

### CODE REVIEW

**Comment by raftaar1191** [comment]
@DevinWalker @ravinderk 

Please review my PR and merge

File: includes/admin/tools/data/class-give-tools-delete-test-transactions.php [hunk 1]
```diff
@@ -235,19 +237,18 @@ public function pre_fetch() {
...
 
-			$posts = get_posts( $args );
-			foreach ( $posts as $post ) {
+			$posts    = new Give_Payments_Query( $args );
+			$payments = $posts->get_payments();
+			foreach ( $payments as $payment ) {
 				$items[] = array(
-					'id'   => (int) $post->ID,
-					'type' => $post->post_type,
+					'id'   => (int) $payment->__get( 'ID' ),
```
Comment (ravinderk) [inline includes/admin/tools/data/class-give-tools-delete-test-transactions.php]: @raftaar1191 Access $ID param directly from payment.
For Ref: https://github.com/WordImpress/Give/commit/0cee706ff28dbcd7fc7e5cdb4c80ab934f90da87

File: includes/admin/tools/data/class-give-tools-delete-test-donors.php [hunk 2]
```diff
@@ -311,9 +311,22 @@ public function get_data() {
 				$this->update_option( $this->step_on_key, '0' );
```
Comment (ravinderk) [inline includes/admin/tools/data/class-give-tools-delete-test-donors.php]: @raftaar1191 When you work any file, Please check other stuff also like documentation,  deprecated or refactor function, code formatting or any logic improvement suggestion.

For Ref: https://github.com/WordImpress/Give/commit/17fa71a64ea32ef7c086b033084002ad49be6d7a

Comment (raftaar1191) [inline includes/admin/tools/data/class-give-tools-delete-test-donors.php]: okay

File: includes/admin/tools/data/class-give-tools-delete-test-transactions.php (hunk 1 above)
Comment (raftaar1191) [inline includes/admin/tools/data/class-give-tools-delete-test-transactions.php]: Okay

**Pull request merged**